### PR TITLE
Fixup build refacto

### PR DESCRIPTION
**Pull request by outscale-avu**

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -132,133 +134,133 @@ echo "### Configuration for packetgraph ###" >> config.mk
...
 if [[ $CC == "" ]];then
-	case $OS in
-		Linux) CC=gcc;;
-		Darwin) CC=clang;;
-		*) CC=cc
-	esac
+    case $OS in
+        Linux) CC=gcc;;
+        Darwin) CC=clang;;
+        *) CC=cc
```
Comment (outscale-avu) [inline configure]: Really usefull?

File: Makefile [hunk 2]
```diff
@@ -2,6 +2,8 @@ include config.mk
 include npf.mk
 include tests.mk
 
+PG_dev_NAME = "$(PG_NAME)-dev"
```
Comment (outscale-mgo) [inline Makefile]: 1: that not related to what the commit explain it does
2: I don't like this variable it's just too much code for something that might never be use

File: Makefile [hunk 3]
```diff
@@ -104,39 +106,3 @@ dev: lpm cdb nvlist thmap qsbr sljit bpfjit npf npfkern $(PG_dev_OBJECTS)
 	ar rcv $(PG_NAME)-dev.a $(PG_dev_OBJECTS) $(lpm_OBJECTS) $(cdb_OBJECTS) $(nvlist_OBJECTS) $(thmap_OBJECTS) $(qsbr_OBJECTS) $(sljit_OBJECTS) $(bpfjit_OBJECTS) $(npf_OBJECTS) $(npfkern_OBJECTS)
 	gcc -shared -Wl,-soname,$(PG_NAME)-dev.so.17 -o $(PG_NAME)-dev.so.17.5.0 $(PG_dev_OBJECTS) $(lpm_OBJECTS) $(cdb_OBJECTS) $(qsbr_OBJECTS) $(sljit_OBJECTS) $(bpfjit_OBJECTS) $(npf_OBJECTS) $(npfkern_OBJECTS) -lc
 	echo "$(PG_CFLAGS)-dev compiled"
-
-clean: clean_npf testcleanobj
```
Comment (outscale-mgo) [inline Makefile]: I think, I don't like that you remove a entire rule from the makefile
I don't like if else in the Makefile too 

Can you keep this rule in the Makefile and just remove the if/else

Thanks,

**Comment by outscale-mgo** [comment]
merge ?

**Pull request closed without merging**